### ISSUE TITLE
ruby 2.4.0 fixnum deprecation warnings

### ISSUE THREAD

**Issue by chrishough** [mattsears/nyan-cat-formatter#74]
```
/Users/chrishough/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/nyan-cat-formatter-0.11/lib/nyan_cat_formatter/rspec3.rb:20: warning: constant ::Fixnum is deprecated
37/37: -_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_,------,
37/37: -_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_|   /\_/\
 0/37: -_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-^|__( - .-)
 0/37: -_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_- " "  " "
```

How do we remove these?

**Comment by mattsears** [comment]
This is fixed in version 0.12

**Comment by chrishough** [comment]
Thank you
